tests/benchmarks/power_consumption/i2c/testcase.yaml

```diff
@@ -0,0 +1,24 @@
+common:
```

Comment:
please add `integration_platforms` section.

Reply:
added for all tests